sandbox/public/index.html

```diff
@@ -200,6 +200,59 @@
         clickCollectionEnabled: false,
       });
     </script>
+
+    <script nonce="%REACT_APP_NONCE%">
+      alloy("configure", {
+        clickCollectionEnabled: true,
+        clickCollection: {
+          internalLinkEnabled: true,
+          externalLinkEnabled: true,
+          downloadLinkEnabled: true,
+          sessionStorageEnabled: false,
+          eventGroupingEnabled: true,
+          filterClickDetails: (props) => {
```

Comment:
This is the new callback. We should remove the console statement but perhaps include a commented out example from the public facing documentation.